docker/Dockerfile

```diff
@@ -38,6 +38,10 @@ ENV DOWNLOAD=${DOWNLOAD}
 
 MAINTAINER FIWARE IoTAgent Team. Telefónica I+D
 
+RUN apt-get update \
```

Comment:
I'd suggest to add the new commands to the main building RUN at L62

Not sure if a matter of style or efficiency (not sure, but each RUN statement is a diferent image layer) but that's the way I understand we typically used in this and other IOTAs. repository.

(Probably a similar comment applies to the other similar PRs in other IOTAs repositories)

Reply:
Fixed in deb48f3 + f5856e8